ledger/sandbox/src/main/scala/com/digitalasset/platform/store/dao/events/TransactionsWriter.scala

```diff
@@ -25,26 +25,32 @@ private[dao] object TransactionsWriter {
       deleteWitnessesBatch: Option[BatchSql],
       insertWitnessesBatch: Option[BatchSql],
   ) {
-    def write()(implicit connection: Connection): Unit = {
-      eventBatches.foreach(_.execute())
+    def write(metrics: Metrics)(implicit connection: Connection): Unit = {
+      import metrics.daml.index.db.storeTransactionDbMetrics
```

Comment:
```suggestion
      val storeTransactionDbMetrics = metrics.daml.index.db.storeTransactionDbMetrics
```
Minor and opinion-based, feel free to not use this.